Extend/Model/Order.php

```diff
@@ -35,29 +36,26 @@
  */
 class Order extends Order_parent
 {
-    const STATE_PENDING = 'pending';
-    const STATE_AUTHORIZED = 'authorized';
-    const STATE_PROCESSING = 'processing';
-    const STATE_CANCELED = 'canceled';
-    const STATE_REFUNDED = 'refunded';
+    // constant strings that are used by several response handlers to handle the specific merchant settings
+    // what should happen if orders are cancelled or failed during the handling of the transaction
```

Comment:
I think the second comment line can be omitted as it is not providing any additional info (clear that it is about canceled or failed status)

Reply:
Also, **canceled** can be spelled with a single "l".

Reply:
👍 

True, in this case 'cancelled' was written like this because the paymentSDK uses this spelling.